pkg/controller/etcdlockserver/reconcile_members.go

```diff
@@ -129,6 +130,7 @@ func memberSpecs(ls *planetscalev2.EtcdLockserver, parentLabels map[string]strin
 		if _, hasClusterLabel := ls.Labels[planetscalev2.ClusterLabel]; hasClusterLabel {
 			labels[planetscalev2.ClusterLabel] = ls.Labels[planetscalev2.ClusterLabel]
 		}
+		update.Labels(&labels, ls.Spec.UserLabels)
```

Comment:
Since we have a new field for this, add this in the append statement below `Annotations` like so:

```
Annotations:       ls.Spec.Annotations,
UserLabels:        ls.Spec.UserLabels,
		})
```

Do the same for the function in reconcile_tablets.go and the vtgate/vtctld deployments.